### DISCUSSION TITLE
Default Hugo build command has drafts enabled

### DISCUSSION THREAD

**Discussion by sophiekoonin** [vercel/community#194] [Help] [answered]
Hi! I just moved my hugo site over from Netlify and realised this morning that when it published, all my draft content had been published too! I looked into it and the default behaviour on Vercel for building hugo apps is `hugo -D --gc`, which is not right at all. The `-D` flag publishes drafts. It should only ever be opt-in and should never be the default value.

![image](https://user-images.githubusercontent.com/10195023/149735646-8b3e3ada-7509-4db4-ac18-1c7ee9eb18f7.png)

I've overridden the command now, but the default value should be `hugo --gc` so as to not surprise anyone by accidentally publishing drafts.

**Comment by RizBizKits** [comment, marked as answer]
Hey @sophiekoonin, thank you for sharing this 💯  I'll make a note of this to convey onto the relevant team via an internal Issue so we can get this sorted!

**Reply by sophiekoonin** [reply]
Thanks so much!
